### ISSUE TITLE
Link to Node, not npm in README.md

### ISSUE THREAD

**Issue by chrisdubya** [niceandserious/crafty-vagrant#16]
I may be wrong about this, but you'll need to have Node installed in order to use npm.  So it might be better to say that you'll need to have Node installed, which comes with npm?

**Comment by nickautomatic** [comment]
Hey Chris. You're right: npm + Gulp do both depend on Node, so Node is also technically a pre-requisite, but I left it out deliberately because I felt that "npm + Gulp" is more specific: there's no way you could have them without also having Node, but I think listing them alone makes it clear at a glance that they're all that Node is needed / used for (while if we listed Node, there are potentially lots of things it could be doing).

Basically, it feels to me that not listing Node shouldn't present any problem to people who are new to npm / Gulp - they'll quickly find that they need Node in order to install them - while making it clear to people who _are_ familiar with Node that this project doesn't use it for anything other than npm / Gulp.

**Comment by chrisdubya** [comment]
Fair enough, I can see how drilling down too much could lead to "Computer with keyboard and mouse" as a requirement.  For some devs who aren't familiar with node, npmjs.com doesn't really make much sense, or give clear installation instructions. nodejs.org does.

If you go to npm on github it says:
" 
## Super Easy Install
npm is bundled with [node](http://nodejs.org/download/).
"

I think that is more clear to someone not familiar with node/npm

**Comment by nickautomatic** [comment]
Yeah, npm's website could definitely do a better job of explaining what npm actually is! But also, part of my rationale was that the set of people who have some familiarity with npm + Gulp is likely to be a lot bigger than the set of people who are familiar with Node (since everyone using Node almost certainly uses npm and at least knows about Gulp).

I think a good general principle is to list direct dependencies but not secondary ones. That keeps the list manageably short, and also avoids making the project look more complex than it is or putting off people who aren't familiar with the indirect dependencies.
